doc/Readme.txt

```diff
@@ -230,6 +230,10 @@ Mac OS X:
 	Get SDL2 and dependencies
 		a) Install "port" from http://www.macports.org/
 		b) sudo port install libsdl2 libsdl2_image libsdl2_mixer
+	or
+		a) Install "homebrew"
+		b) sudo brew install libsdl2 libsdl2_image
+		c) sudo brew install sdl2_mixer --with-libvorbis
```

Comment:
Please no `sudo`, homebrew was designed to not need it, see https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/FAQ.md#why-does-homebrew-say-sudo-is-bad-

> Homebrew is designed to work without using sudo. You can decide to use it but we strongly recommend not to do so. If you have used sudo and run into a bug then it is likely to be the cause.

Thanks :-)